content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,87 @@
+# Persisting Experiments
+
+DVC runs experiments outside of the standard Git stage/commit workflow for quick
+iteration. In this section we'll see how to bring the experiments we run to the
+standard Git workflow with `dvc exp branch` and `dvc exp apply`.
+
+## Create a Git branch for your experiment
+
+You can use `dvc exp branch` to create a new branch from the experiment and keep
+all code and artifacts separate from your current one.
+
+```dvc
+$ dvc exp show --include-params=my_param
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃     metric:**auc** ┃ param:**my_param**   ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━┩
+│ workspace             │ -            │ 0.61314 │ 3          │
+│ experiments-tag       │ Oct 19, 2020 │ 0.61314 │ 3          │
+│ ├── exp-e6c97         │ Oct 20, 2020 │ 0.69830 │ 2          │
+│ └── exp-1df77         │ Oct 22, 2020 │ 0.51676 │ 1          │
+└───────────────────────┴──────────────┴─────────┴────────────┘
+```
+
+Suppose you want to continue to work on `exp-e6c97` in a separate branch. You
+can create a new Git branch by specifying the experiment and giving a new name
+for it:
+
+```dvc
+$ dvc exp branch exp-e6c97 my-branch
+Git branch 'my-branch' has been created from experiment 'exp-e6c97'.
+To switch to the new branch run:
+        git checkout my-branch
+```
+
+Note that DVC doesn't checkout the new branch. You can one or more branches from
+existing experiments, and switch into any one manually like this:
+
+```dvc
+$ git checkout my-branch
+$ dvc checkout
+```
+
+Your workspace now contains all the files from the experiment.
+
+## Bring Experiment results to your workspace
+
+Typically, `dvc exp run` leaves the experiment results in your workspace for
+convenience. However, you may have run multiple experiments and wish to go back
+to a specific one. In this case, you can restore a previous experiment's results
+with `dvc exp apply`. Let's see an example:
+
+```dvc
+$ dvc exp show --include-params=my_param
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃     metric:**auc** ┃ param:**my_param**   ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━┩
+│ workspace             │ -            │ 0.61314 │ 3          │
+│ experiments-tag       │ Oct 19, 2020 │ 0.61314 │ 3          │
+│ ├── exp-e6c97         │ Oct 20, 2020 │ 0.69830 │ 2          │
+│ └── exp-1df77         │ Oct 22, 2020 │ 0.51676 │ 1          │g
+└───────────────────────┴──────────────┴─────────┴────────────┘
+```
+
+The results found in the workspace can be found in the respective row. When you
+want to bring another experiment to the workspace, you can refer to it using
+it's name or ID, e.g.:
+
+```dvc
+$ dvc exp apply exp-e6c97
+Changes for experiment 'exp-e6c97' have been applied...
+```
```

Comment:
We may want to note that the working tree has to be clean (stash or commit changes).

Reply:
> > the working tree has to be clean
>
> there is no such restriction

OK maybe not a hard restriction (not always), but `exp apply` will fail if there are conflicts between the experiment results and a dirty working tree, right?